zeppelin-web/src/app/notebook/notebook-actionBar.html

```diff
@@ -126,11 +126,20 @@
 
 <!-- put the delete action by itself for your protection -->
       <span class="labelBtn" style="vertical-align:middle; display:inline-block;">
-        <button type="button"
+        <button ng-if="note.name.split('/')[0] === '~Trash'"
```

Comment:
How about creating a function for this comparison into a "controller.js"?
And it would be nice to make a `const` value for `~Trash`!